examples/pytorch/graphsage/multi_gpu_node_classification.py

```diff
@@ -69,13 +69,14 @@ def inference(self, g, device, batch_size):
                 batch_size=batch_size, shuffle=False, drop_last=False,
                 num_workers=0, use_ddp=True, use_uva=True)
 
+        last_nd = None
```

Comment:
@BarclayII could we hide this in the `dataloader` object above so there is no explicit `nd` in user's code?

Reply:
It's hard because `y` is outside the DataLoader (L79 and L89).

Reply:
This is really an advanced usage of pinned tensors in dataloader. It updates the feature and reuses a pre-defined dataloader for UVA feature fetching. 
In common cases, things will be much easier because we seldom modify the pinned tensors in the dataloader.

Reply:
Could we re-initialize the DataLoader for every layer?

Reply:
Hided `nd` by re-initializing the dataloader for every layer.